doc/Extensions/Services.md

```diff
@@ -3,8 +3,8 @@ source: Extensions/Services.md
 
 Services within LibreNMS provides the ability to use Nagios plugins to perform additional monitoring outside of SNMP.
 
-These services are tied into an existing device so you need at least one device that supports SNMP to be able to add it
-to LibreNMS - localhost is a good one.
+**These services are tied into an existing device so you need at least one device that supports SNMP to be able to add it
```

Comment:
It doesn't need to support snmp any more.

Reply:
Yes but the service check has to be attached to a device that is in librenms in order for the alerting to work. Also how sure are you on the SNMP? https://community.librenms.org/t/service-check-alert/2954/25

Reply:
well crap, looks like we need to update the code to allow that!

Reply:
@kkrumm1 I could not reproduce the need for snmp...

Reply:
Yeah, I saw the post... I was just going off the word of amaizenshtein.